### DISCUSSION TITLE
How to predict the next value when the model has been evaluated

### DISCUSSION THREAD

**Discussion by oscarstv** [online-ml/river#1489] [Q&A] [answered]
Hello.
I have used the airline-passengers.csv model you can find in the documentation.
Last value is 1960-12.

How can I predict 1961-01 value?
Using model.predict_one(x)
I get 71.54166580436763, which I think is incorrect.

**Comment by smastelini** [comment]
I suggest you check the [example](https://riverml.xyz/latest/examples/building-a-simple-nowcasting-model/) about creating a nowcasting model in the documentation.

**Reply by oscarstv** [reply]
Of course this is what I did.
It's all done in Jupyter notebook and works as expected.

And now I would like to predict one more value, one more, beyond the last value of the file airline-passengers.csv

month passengers
1949-01 112
...
...
1960-12 432

**Reply by smastelini** [reply, marked as answer]
You will need to assemble the needed info (`x` values) yourself, to pass to the model. There are multiple strategies to extrapolate prediction in the future. For instance, you can feed the model its predictions to predict possibly unbounded forecasting horizons. This is what is called a recursive forecasting approach. There are drawbacks, of course, such as error propagation.
